lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -372,6 +372,13 @@ def self.get_name(instance_type, api_name, is_reserved = false)
       't2.small' => :low_to_moderate,
       't2.xlarge' => :high,   # same as c4.2xlarge, cf:https://aws.amazon.com/blogs/aws/new-t2-xlarge-and-t2-2xlarge-instances/
       't2.2xlarge' => :high,  # same as m4.xlarge,  cf:https://aws.amazon.com/blogs/aws/new-t2-xlarge-and-t2-2xlarge-instances/
+      't3.nano' => :low,
```

Comment:
where are the values here acquired?

Reply:
Pulled these directly from the pricing json. There's a section for network performance:

ex)
    "Y6TJBM7SJGY7QRS4" : {
      "sku" : "Y6TJBM7SJGY7QRS4",
      "productFamily" : "Compute Instance",
      "attributes" : {
        "servicecode" : "AmazonEC2",
        "location" : "Asia Pacific (Sydney)",
        "locationType" : "AWS Region",
        "instanceType" : "t3.nano",
        "currentGeneration" : "Yes",
        "instanceFamily" : "General purpose",
        "vcpu" : "2",
        "physicalProcessor" : "Intel Skylake E5 2686 v5 (2.5 GHz)",
        "memory" : "0.5 GiB",
        "storage" : "EBS only",
        "networkPerformance" : "Low",
        "processorArchitecture" : "64-bit",
        "tenancy" : "Shared",
        "operatingSystem" : "SUSE",
        "licenseModel" : "No License required",
        "usagetype" : "APS2-BoxUsage:t3.nano",
        "operation" : "RunInstances:000g",
        "capacitystatus" : "Used",
        "ecu" : "Variable",
        "normalizationSizeFactor" : "0.25",
        "preInstalledSw" : "NA",
        "servicename" : "Amazon Elastic Compute Cloud"
      }
    },

Attributes -> Network performance